docs/user/openstack/customization.md

```diff
@@ -1,29 +1,55 @@
 # OpenStack Platform Customization
 
-In the OpenShift Installer `install-config.yaml` you can set the following options regarding the OpenStack platform:
+Beyond the [platform-agnostic `install-config.yaml` properties](../customization.md#platform-customization), the installer supports additional, OpenStack-specific properties.
 
-- `machines.platform.openstack.region` - The OpenStack region where the cluster will get created
-- `machines.platform.openstack.cloud` - Name of the OpenStack cloud to use from clouds.yaml
-- `machines.platform.openstack.externalNetwork` - The OpenStack external network name to be used for installation
-- `machines.platform.openstack.computeFlavor` - The OpenStack compute flavor to use for master servers
-- `machines.platform.openstack.lbFloatingIP` - Existing Floating IP to associate with API loadbalancer
-- `machines.platform.openstack.trunkSupport` - Whether OpenStack ports can be trunked. True or False
-- `machines.platform.openstack.octaviaSupport` - Whether OpenStack supports Octavia. True of False
-- `machines.platform.openstack.defaultMachinePlatform` - (optional) The default configuration used when installing on OpenStack for machine pools
+## Cluster-scoped properties
 
-For more technical definitions, see the [go docs](https://godoc.org/github.com/openshift/installer/pkg/types/openstack#Platform).
+* `cloud` (required string): The name of the OpenStack cloud to use from `clouds.yaml`.
+* `computeFlavor` (required? string): The OpenStack compute flavor to use for master servers (FIXME: how is this different from `controlPlane.platform.openstack.type`?).
```

Comment:
It's also strange that we use `compute` in a parameter that is configuring the control plane machines.